graphql/typeDefs/User/query.js

```diff
@@ -1,9 +1,6 @@
+// TODO(Bouncey): requires auth protection
 export default `
 type Query {
-  users(
-      _id: ID
-      name: String
-      email: String
-  ): [User]
+  getSessionUser(email: String!): User
```

Comment:
I'm assuming you created this to get the user object for the current user provided the session is authenticated? How about letting user() returning the currently logged in user, and returning an empty result if here is no user authenticated?

The following query:

```graphql
query {
  user {
    id
  }
}
```

Would then return, when authenticated:

```graphql
{
  "data": {
    "user": {
      "id": "e82403ce-4687-11e8-bca3-14109fd1f8cb" 
    }
  }
}
```

Or, when not authenticated, an empty user, with a helpful error object:

```graphql
{
  "data": {
      "user": null
    },
   "errors": {
        "message": "Not logged in",
        "locations": [
            {
                "line": 2,
                "column": 2
        ],
        "path": [
            "user"
        ]
    }
}
```